aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/TccCommands/ComputeTransactions.java

```diff
@@ -0,0 +1,54 @@
+package com.castsoftware.aip.console.tools.commands.TccCommands;
+
+import com.castsoftware.aip.console.tools.commands.BasicCallable;
+import com.castsoftware.aip.console.tools.commands.SharedOptions;
+import com.castsoftware.aip.console.tools.core.dto.tcc.ComputeFunctionPointsProperties;
+import com.castsoftware.aip.console.tools.core.services.ApplicationService;
+import com.castsoftware.aip.console.tools.core.services.JobsService;
+import com.castsoftware.aip.console.tools.core.services.RestApiService;
+import com.castsoftware.aip.console.tools.core.utils.VersionInformation;
+import com.castsoftware.aip.console.tools.providers.CliLogPollingProviderImpl;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import picocli.CommandLine;
+
+@Component
+@CommandLine.Command(name = "compute-transactions", mixinStandardHelpOptions = true, description = "Compute the transactions for the specified application.")
+@Slf4j
+@Getter
+@Setter
+public class ComputeTransactions extends BasicCallable {
+    @CommandLine.ParentCommand
+    private TccCommand parentCommand;
+
+    @CommandLine.Option(names = "--wait", paramLabel = "WAIT", description = "Wait for the compute to finish.", negatable = true, defaultValue = "true", fallbackValue = "true")
+    boolean wait;
+
+    public ComputeTransactions(RestApiService restApiService, JobsService jobsService, ApplicationService applicationService) {
+        super(restApiService, jobsService, applicationService);
+    }
+
+    @Override
+    public Integer processCallCommand() throws Exception {
+        ComputeFunctionPointsProperties computeFunctionPointsProperties = ComputeFunctionPointsProperties.builder()
+                .applicationName(parentCommand.getApplicationName())
+                .wait(wait)
```

Comment:
Because some sub commands don't need wait. There are sub commands like 'list-rules', 'check-content', 'check-all-content' etc which don't care about wait because these actions are not jobs.

Reply:
Ok got it
Only few sub commands required that